systemd/docker-lvm-plugin.service

```diff
@@ -5,8 +5,8 @@ Before=docker.service
 
 [Service]
 Type=simple
-EnvironmentFile=-/etc/sysconfig/docker-lvm-volumegroup
-ExecStart=/usr/bin/docker-lvm-plugin
+EnvironmentFile=-/etc/sysconfig/docker-lvm-plugin
```

Comment:
I like it under sysconfig like the other files such as docker,docker-storage-setup and the like